k8s/helm-charts/seldon-core-v2-setup/values.yaml.template

```diff
@@ -252,7 +252,7 @@ serverConfig:
       pullPolicy: IfNotPresent
       registry: docker.io
       repository: seldonio/mlserver
-      tag: 1.3.5
+      tag: 1.5.0
```

Comment:
without this change, on automated re-generation of helm-charts & yamls, 1.3.5 was being introduced instead of 1.5.0

Reply:
nice catch